src/web/components/SlotBodyEnd.tsx

```diff
@@ -102,7 +102,7 @@ const MemoisedInner = (props: Props) => {
 
     useEffect(() => {
         const contributionsPayload = buildPayload(props);
-        getBodyEnd(contributionsPayload)
+        getBodyEnd(contributionsPayload, 'http://localhost:8081/epic')
```

Comment:
Just adding a reminder that we don't want to accidentally merge this change :-)

Reply:
That's right 😬